packages/jest-runtime/src/index.ts

```diff
@@ -162,6 +162,7 @@ class Runtime {
   private _isolatedModuleRegistry: ModuleRegistry | null;
   private _moduleRegistry: ModuleRegistry;
   private _esmoduleRegistry: Map<string, Promise<VMModule>>;
+  private _path: Config.Path;
```

Comment:
call it `testPath` or something like that

Reply:
Done!